starter-code/src/functions-and-arrays.js

```diff
@@ -12,6 +35,23 @@ var words = [
 ];
 
 // Calculating a Sum
+function sumArray(array) {
+  var sum = 0;
+  array.forEach(function (element) {
+    sum += element;
+  });
+  return sum;
+};
+
+function averageNumbers(array) {
+  if (!array.length) {
+    return;
+  } else if (array.length === 1) {
+    return array[0];
```

Comment:
You don't need this step ```sumArray()``` of 1 element in the array divided by ```array.length = 1``` will be the same as next step. Same as we talked about in findLongestWord exercise.